tests/client/handlers/nodes.test.ts

```diff
@@ -680,3 +694,266 @@ describe('nodesPing', () => {
     );
   });
 });
+describe('nodesGetAll', () => {
+  const logger = new Logger('nodesPing test', LogLevel.WARN, [
+    new StreamHandler(
```

Comment:
Update the logger name to reflect  what you're actually testing. Same for the other one.